src/test/java/ca/on/oicr/gsi/dimsum/service/filtering/QCStatusSortTest.java

```diff
@@ -0,0 +1,26 @@
+package ca.on.oicr.gsi.dimsum.service.filtering;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class QCStatusSortTest {
+
+  public List<String> QC_STATUS_ORDER =
+      Arrays.asList("pendingQC", "dataReview", "topUp", "passed", "failed");
+
+  @Test
+  public void testSortByStatusAscending() {
+    QCStatusSort qcStatusSort = new QCStatusSort();
+    qcStatusSort.setReverseOrder(false);
+    Collections.sort(QC_STATUS_ORDER, Collections.reverseOrder());
+  }
+
+  @Test
+  public void testSortByStatusDescending() {
+    QCStatusSort qcStatusSort = new QCStatusSort();
+    qcStatusSort.setReverseOrder(true);
+    Collections.sort(QC_STATUS_ORDER);
+  }
```

Comment:
* These tests are not actually using `QCStatusSort` to sort `QC_STATUS_ORDER`, and they couldn't because `QCStatusSort` compares Samples and `QC_STATUS_ORDER` contains Strings. `Collections#sort` and `Collections.reverseOrder` both use natural order - not `QCStatusSort`
* The tests are not making any assertions to check whether the correct results were achieved
* Instead of this separate test class, please add tests to `SampleSortTest` that test sorting Samples using `SampleSort.QC_STATUS`